x-pack/plugins/transform/public/app/sections/transform_management/components/action_delete/delete_action_modal.tsx

```diff
@@ -47,21 +47,6 @@ export const DeleteActionModal: FC<DeleteAction> = ({
   });
   const bulkDeleteModalContent = (
     <>
-      <p>
```

Comment:
There are two import statements at the top of this file which need to be removed to get the CI type check test to pass:

```
import { FormattedMessage } from '@kbn/i18n/react';
import { TRANSFORM_STATE } from '../../../../../../common';
```